person.rb

```diff
@@ -1,4 +1,10 @@
-class Person
+class Nameable
```

Comment:
- [ ] So, in the previous milestone, you were required to place your classes in a separate file, but your code reviewer approved your code without mentioning that! 

![image](https://user-images.githubusercontent.com/69011963/216070168-194240c0-eda7-4296-8dab-de6d52b7f85b.png)

In this milestone, I encourage you to place your classes in a separate file. Except for the Person class places other classes into separate `.rb` files. Thank you!

Reply:
@Rustamxon7 In previous Milestone i put everything in separate file but in this milestone Microvese didn't mention it.